src/course-unit/data/thunk.js

```diff
@@ -303,3 +305,20 @@ export function setXBlockOrderListQuery(blockId, xblockListIds, restoreCallback)
     }
   };
 }
+
+export function fetchXBlockIframeHtmlAndResourcesQuery(xblockId) {
+  return async (dispatch) => {
+    dispatch(updateSavingStatus({ status: RequestStatus.PENDING }));
+    dispatch(showProcessingNotification(NOTIFICATION_MESSAGES.adding));
+
+    try {
```

Comment:
```suggestion
    dispatch(showProcessingNotification(NOTIFICATION_MESSAGES.adding));
    try {
```

Reply:
We have this indentation in all sleds in this file
Do we need to delete it specifically in `fetchXBlockIframeHtmlAndResourcesQuery`?